hunsri/EwMM-ProjectCS
# Ft/main menu data

- Integrate `MainMenu` with `PlayerData` and `PlayerDataManager`
- No longer using `PlayerData` component inside the `SettingsScript` folder

### How to test
- Go to MainMenu scene and play around with the slider.
- Restart game
- State should persist

Keep in mind that state wouldn't persist between Scenes (yet)!
